components/DetailsHeader/DetailsHeader.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <div class="details-header">
+    <div class="details-header__page-route">
+      <p>
+        <nuxt-link
+          :to="{
+            name: breadcrumb.name,
+            query: {
+              type: breadcrumb.type
+            }
+          }"
+        >
+          {{ breadcrumb.parent }}
+        </nuxt-link>
+        > {{ formatBreadcrumb(data.title) }}
+      </p>
+    </div>
+    <div class="details-header__container container">
+      <div class="details-header__container--image">
+        <slot name="banner image" />
+      </div>
+      <div class="details-header__container--content">
+        <h3>
+          {{ data.organ.fields.name }}
```

Comment:
For the `DetailsHeader`, the props should be simple, context agnostic props such as `title`, `subheading`, `descrption`, etc. As is, it is only for a specific content type. Allow the page that has `DetailsHeader` (like the projects landing page) to define this data and pass it as a prop to `DetailsHeader`.

Reply:
Yeah that's what I was thinking because I ran into this issue with the organ page. I'll fix.